source/pug/partials/pulse-projects.pug

```diff
@@ -12,8 +11,9 @@ mixin projects(name, description, url, projectData)
 
 mixin project(meta)
   .col-sm-12.col-md-6.mb-4.mb-md-0
-    img.project-image(src=meta.thumbnail)
-    h4.medium-black.my-2=meta.title
+    a(href=`https://www.mozillapulse.org/entry/${meta.id}`)
```

Comment:
Pulse front end is hard coded here but the API is a variable. I can see that causing trouble. Can we set this as a variable in the same spot the others are declared, at the least?